### PR TITLE
chore(nextJS): updated nextjs getting-started with accurate cli prompts

### LINKED ISSUE #5206: [Feedback]NextJS getting-started instructions missing CLI commands
**Page**: [`/start/getting-started/data-model/q/integration/[integration]`](https://docs.amplify.aws/start/getting-started/data-model/q/integration/next)

**Feedback**:

The instructions only cover amplify add api, yet the push command shows that an auth has been run as well. Please add the amplify add auth details.

### LINKED ISSUE #5366: [Feedback]Next.js getting started guide has outdated CLI instructions
**Page**: [`/start/getting-started/data-model/q/integration/[integration]`](https://docs.amplify.aws/start/getting-started/data-model/q/integration/next)

**Feedback**:

### What it shows:
```console
? Select from one of the below mentioned services: (GraphQL)

? Here is the GraphQL API that we will create. Select a setting to edit or continue
Authorization modes: (API key (default, expiration time: 7 days from now))

? Choose the default authorization type for the API: (Amazon Cognito User Pool)

Using service: Cognito, provided by: awscloudformation
The current configured provider is Amazon Cognito.

? Do you want to use the default authentication and security configuration? Default configuration: (Default configuration) 

Warning: you will not be able to edit these selections.

? How do you want users to be able to sign in? (Username)
? Do you want to configure advanced settings? No, I am done. (No, I am done.)

✅ Successfully added auth resource nextamplified locally

✅ Some next steps:
"amplify push" will build all your local backend resources and provision it in the cloud
"amplify publish" will build all your local backend and frontend resources (if you have hosting category added) and provision it in the cloud

? Configure additional auth types? (No)
? Here is the GraphQL API that we will create. Select a setting to edit or continue: (Continue)
? Choose a schema template: (Single object with fields (e.g., “Todo” with ID, name, description))

# You should see that the GraphQL schema compiled successfully and the location for the file. 

? Do you want to edit the schema now? (Yes)
```

### What I saw when I ran through the process:
```console
amplify add api
? Select from one of the below mentioned services: GraphQL
? Here is the GraphQL API that we will create. Select a setting to edit or continue Cont
inue
? Choose a schema template: Single object with fields (e.g., “Todo” with ID, name, descr
iption)

⚠️  WARNING: your GraphQL API currently allows public create, read, update, and delete access to all models via an API Key. To configure PRODUCTION-READY authorization rules, review: https://docs.amplify.aws/cli/graphql/authorization-rules

✅ GraphQL schema compiled successfully.

Edit your schema at /Users/<user>/Desktop/<project-path>/next-amplified/amplify/backend/api/nextamplified/schema.graphql or place .graphql files in a directory at /Users/<user>/Desktop/<project-path>/next-amplified/amplify/backend/api/nextamplified/schema
✔ Do you want to edit the schema now? (Y/n) · yes
Edit the file in your editor: /Users/<user>/Desktop/<project-path>/next-amplified/amplify/backend/api/nextamplified/schema.graphql
✅ Successfully added resource nextamplified locally

✅ Some next steps:
"amplify push" will build all your local backend resources and provision it in the cloud
"amplify publish" will build all your local backend and frontend resources (if you have hosting category added) and provision it in the cloud
```

### Other Notes
This also no longer applies to the CLI:
image
Instead, the user should save the file in their editor before continuing on to the next step

### What it shows:
```console
Current Environment: dev

┌──────────┬───────────────────────┬───────────┬───────────────────┐
│ Category │ Resource name         │ Operation │ Provider plugin   │
├──────────┼───────────────────────┼───────────┼───────────────────┤
│ Auth     │ nextamplifiedXXXXXXX  │ Create    │ awscloudformation │
├──────────┼───────────────────────┼───────────┼───────────────────┤
│ Api      │ nextamplified         │ Create    │ awscloudformation │
└──────────┴───────────────────────┴───────────┴───────────────────┘
? Are you sure you want to continue? Yes

# You will be walked through the following questions for GraphQL code generation


API key configuration
✔ Enter a description for the API key: · (default)
✔ After how many days from now the API key should expire (1-365): · 7 (default)

# You should see that the GraphQL schema compiled successfully.

? Do you want to generate code for your newly created GraphQL API: (Yes)
? Choose the code generation language target: (javascript)
? Enter the file name pattern of graphql queries, mutations and subscriptions (src/graphql/**/*.js) (default)
? Do you want to generate/update all possible GraphQL operations - queries, mutations and subscriptions: (Yes)
? Enter maximum statement depth [increase from default if your schema is deeply nested]: (2)

# Wait a minute or so for the deployment into the dev environment to complete
...

✔ Generated GraphQL operations successfully and saved at src/graphql
✔ All resources are updated in the cloud
```

### What I saw when I ran through the process:
```console
amplify push
⠧ Fetching updates to backend environment: dev from the cloud.⠋ Building resource api/ne⠙ Fetching updates to backend environment: dev from the cloud.
⚠️  WARNING: your GraphQL API currently allows public create, read, update, and delete access to all models via an API Key. To configure PRODUCTION-READY authorization rules, review: https://docs.amplify.aws/cli/graphql/authorization-rules

✔ Successfully pulled backend environment dev from the cloud.
⠹ Building resource api/nextamplified
⚠️  WARNING: your GraphQL API currently allows public create, read, update, and delete access to all models via an API Key. To configure PRODUCTION-READY authorization rules, review: https://docs.amplify.aws/cli/graphql/authorization-rules


    Current Environment: dev
    
┌──────────┬───────────────┬───────────┬───────────────────┐
│ Category │ Resource name │ Operation │ Provider plugin   │
├──────────┼───────────────┼───────────┼───────────────────┤
│ Api      │ nextamplified │ Create    │ awscloudformation │
└──────────┴───────────────┴───────────┴───────────────────┘
✔ Are you sure you want to continue? (Y/n) · yes

⚠️  WARNING: your GraphQL API currently allows public create, read, update, and delete access to all models via an API Key. To configure PRODUCTION-READY authorization rules, review: https://docs.amplify.aws/cli/graphql/authorization-rules

Cognito UserPool configuration
Using service: Cognito, provided by: awscloudformation
 
 The current configured provider is Amazon Cognito. 
 
 Do you want to use the default authentication and security configuration? Default confi
guration
 Warning: you will not be able to edit these selections. 
 How do you want users to be able to sign in? Username
 Do you want to configure advanced settings? No, I am done.
✅ Successfully added auth resource nextamplifiedccf618e2 locally

✅ Some next steps:
"amplify push" will build all your local backend resources and provision it in the cloud
"amplify publish" will build all your local backend and frontend resources (if you have hosting category added) and provision it in the cloud


⚠️  WARNING: your GraphQL API currently allows public create, read, update, and delete access to all models via an API Key. To configure PRODUCTION-READY authorization rules, review: https://docs.amplify.aws/cli/graphql/authorization-rules

⚠️ WARNING: owners may reassign ownership for the following model(s) and role(s): Post: [owner]. If this is not intentional, you may want to apply field-level authorization rules to these fields. To read more: https://docs.amplify.aws/cli/graphql/authorization-rules/#per-user--owner-based-data-access.
✅ GraphQL schema compiled successfully.

Edit your schema at /Users/<user>/Desktop/<project-path>/next-amplified/amplify/backend/api/nextamplified/schema.graphql or place .graphql files in a directory at /Users/<user>/Desktop/<project-path>/next-amplified/amplify/backend/api/nextamplified/schema

? Do you want to generate code for your newly created GraphQL API Yes
? Choose the code generation language target javascript
? Enter the file name pattern of graphql queries, mutations and subscriptions src/graphq
l/**/*.js
? Do you want to generate/update all possible GraphQL operations - queries, mutations an
d subscriptions Yes
? Enter maximum statement depth [increase from default if your schema is deeply nested] 
2

```

### PR DESCRIPTION
**Pull request by tannerabread**
#### Related GitHub issue #, if available:
Closes #5366, resolves #5206

### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?
- [ ] amplify-cli
- [ ] amplify-ui
- [ ] amplify-studio
- [ ] amplify-hosting
- [x] amplify-libraries

Which platform(s) are affected by this PR (if applicable)?
- [x] JS
- [ ] iOS
- [ ] Android
- [ ] Flutter
- [ ] React Native

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks

- [x] Does this PR conform to [the styleguide](https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md)?

- [ ] Does this PR include filetypes other than markdown or images? Please add or update unit tests accordingly.

- [ ] Are any files being deleted with this PR? If so, have the needed redirects been created?

- [ ] Are all links in MDX files using the MDX link syntax rather than HTML link syntax? <br /> 
      _ref: MDX: `[link](https://link.com)` 
            HTML: `<a href="https://link.com">link</a>`_

### When this PR is ready to merge, please check the box below
- [ ] Ready to merge

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

**Review by abdallahshaban557 — APPROVED** [review]
LGTM

**Review by jimblanc — DISMISSED** [review]

File: src/fragments/start/getting-started/next/api.mdx [hunk 1]
```diff
@@ -79,38 +63,47 @@ amplify push
...
-? Choose the code generation language target: (javascript)
-? Enter the file name pattern of graphql queries, mutations and subscriptions (src/graphql/**/*.js) (default)
-? Do you want to generate/update all possible GraphQL operations - queries, mutations and subscriptions: (Yes)
-? Enter maximum statement depth [increase from default if your schema is deeply nested]: (2)
+⚠️ WARNING: owners may reassign ownership for the following model(s) and role(s): Post: [owner]. If this is not intentional, you may want to apply field-level authorization rules to these fields. To read more: https://docs.amplify.aws/cli/graphql/authorization-rules/#per-user--owner-based-data-access.
+✅ GraphQL schema compiled successfully.
 
-# Wait a minute or so for the deployment into the dev environment to complete
-...
+Edit your schema at /Users/<user>/Desktop/<project-path>/next-amplified/amplify/backend/api/nextamplified/schema.graphql or place .graphql files in a directory at /Users/<user>/Desktop/<project-path>/next-amplified/amplify/backend/api/nextamplified/schema
```
[the remark this answers is not in the document]
Comment (katiegoines) [reply src/fragments/start/getting-started/next/api.mdx:100]: @josefaidt @rachelnabors @abdallahshaban557 - Is this a blocker for this PR or should it be addressed separately? Thanks!

[the remark this answers is not in the document]
Comment (abdallahshaban557) [reply src/fragments/start/getting-started/next/api.mdx:100]: I think we should change it to "<path-to-project>/next-amplified/amplify/backend/api/nextamplified/schema.graphql" as Josif recommended. FYI @tannerabread

[the remark this answers is not in the document]
Comment (tannerabread) [reply src/fragments/start/getting-started/next/api.mdx]: changed in 4-5 locations to the suggested `<path-to-project>`

**Review by abdallahshaban557 — APPROVED** [review]
LGTM

**Review by katiegoines — APPROVED** [review]

**Pull request merged**